aws_lambda_builders/workflows/custom_make/workflow.py

```diff
@@ -63,18 +60,12 @@ def __init__(self, source_dir, artifacts_dir, scratch_dir, manifest_path, runtim
 
         self.actions = []
 
-        if not self.build_in_source:
-            # if we're building on scratch_dir, we have to first copy the source there
-            self.actions.append(CopySourceAction(source_dir, scratch_dir, excludes=self.EXCLUDED_FILES))
+        if working_directory != source_dir:
+            # if we're not building in the source directory, we have to first copy the source
+            self.actions.append(CopySourceAction(source_dir, working_directory, excludes=self.EXCLUDED_FILES))
 
         self.actions.append(make_action)
 
-    def _select_working_directory(self, source_dir: str, scratch_dir: str, build_in_source: bool):
```

Comment:
Due to the changes in this PR, we can remove the logic of choosing a working/build directory based on the `build_in_source` parameter. Similarly, in other workflows we won't have to define this kind of logic either because the parent Workflow class now handles setting the `build_dir`.